app/pages/custom-realm-dashboard.tsx

```diff
@@ -202,18 +234,31 @@ function CustomRealmDashboard({ alert }: Props) {
       enableColumnFilter: false,
       enableSorting: false,
       cell: (props: any) => {
-        const disabled = props.row.original.status !== 'applied' || props.row.original.archived === true;
+        const deleteDisabled = props.row.original.status !== 'applied' || props.row.original.archived === true;
+        const restoreDisabled =
+          ![StatusEnum.APPLIED, StatusEnum.PRSUCCESS].includes(props.row.original.status) ||
```

Comment:
should this be and instead of or?

Reply:
this is intentional, the restore should always be disabled if archived is false since it isn't deleted regardless of status